css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-font-size-xs: $font-size-9 !default;
```

Comment:
I do not think we need a 12px avatar.

Reply:
@wibjorn  So the avatar sizes we  use on Learn are `32`, `36`, and `48` with corresponding font sizes `14px`, `16px` and `20px`:

![image](https://github.com/microsoft/atlas-design/assets/57374379/eec02fb4-e758-4bf2-b4d4-eb6b46acf70f)

But per Figma's conversation, you were already asking to shrink it down to 3, but Courtney's feedback was to stick to 5, so that's why I made these - `28/12px`, `32/14px`, `36/16px`, `42/18px`, and `48/20px`, with 12px being the smallest to pass the accessibility reqs

![image](https://github.com/microsoft/atlas-design/assets/57374379/0b68deaa-0cec-45a3-b400-81863876a3e2)

Let me know if you feel like cutting those we don't need at the moment is okay and I'll do it.

Also, worth mentioning, in Figma avatar sizes are `16`, `24`, `28`, `32`, and `40` with the first two being tiny and initials won't be readable, so I cut those and added a few more to satisfy the request of having 5 sizes and included `36` and `48` we need.

PS I'll find a designer to update Figma to match what we'll end up with.

Reply:
Updated to 3 sizes.